recipes/recastnavigation/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+
+
+class RecastNavigationConan(ConanFile):
+    name = "recastnavigation"
+    homepage = "https://github.com/recastnavigation/recastnavigation"
+    description = " Navigation-mesh Toolset for Games"
+    topics = ("conan", "navmesh", "recast", "navigation", "crowd")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "ZLIB"
```

Comment:
```suggestion
    license = "Zlib"
```

https://spdx.org/licenses/Zlib.html